gremlin-test/src/main/java/org/apache/tinkerpop/gremlin/structure/io/IoTest.java

```diff
@@ -721,14 +732,18 @@ public static void assertCrewGraph(final Graph g1, final boolean lossyForId) {
     }
 
     public static void assertClassicGraph(final Graph g1, final boolean assertDouble, final boolean lossyForId) {
-        assertToyGraph(g1, assertDouble, lossyForId, false);
+        assertToyGraph(g1, assertDouble, lossyForId, false, true);
+    }
+
+    public static void assertNoEdgeGraph(final Graph g1, final boolean assertDouble, final boolean lossyForId) {
+        assertToyGraph(g1, assertDouble, lossyForId, false, false);
     }
 
     public static void assertModernGraph(final Graph g1, final boolean assertDouble, final boolean lossyForId) {
-        assertToyGraph(g1, assertDouble, lossyForId, true);
+        assertToyGraph(g1, assertDouble, lossyForId, true, true);
     }
 
-    private static void assertToyGraph(final Graph g1, final boolean assertDouble, final boolean lossyForId, final boolean assertSpecificLabel) {
+    private static void assertToyGraph(final Graph g1, final boolean assertDouble, final boolean lossyForId, final boolean assertSpecificLabel, final boolean assertEdgeLabel) {
```

Comment:
right - but you've modified a core piece of code with `assertToyGraph` by adding logic that really doesn't generally apply to anything else that would use that logic besides `shouldReadGraphMLWithNoEdgeLabels` which is testing a version of "toy graph" that is not even a legitimate representation of our toy graph because it has no labels. 

I would prefer that you simplify your `shouldReadGraphMLWithNoEdgeLabels` test to read some graphml with just two vertices and an edge with no label and then assert that entirely within that test method. Then your test and changes are isolated specifically to just `shouldReadGraphMLWithNoEdgeLabels` and that core logic remains unchanged.